### PR TITLE
Slue PR configs

### PR DESCRIPTION
**Pull request by siddhu001**
Upload configs and provide access to best pre-trained models for SLT paper (https://arxiv.org/abs/2211.05869).

### CODE REVIEW

**Comment by siddhu001** [comment]
Hi @ftshijt,

This PR is ready for review. The primary modifications include the addition of configuration files for the best-trained model in the SLT paper and the ESPnet-SLUE toolkit, which was presented at the ASRU workshop. I've also implemented some minor fixes to the scoring function and the 2-pass SLU implementation. Could you please review it?

**Comment by siddhu001** [comment]
Hi @sw005320,

This PR is related to slue-voxceleb and slue-voxpopuli recipes for ESPnet-SLUE toolkit. Please let me know if you have any comments before we can merge this PR!

**Comment by sw005320** [comment]
OK, I'll start to review.
@ftshijt, can you also start it?

File: egs2/slue-voxpopuli/slu1/run.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="devel"
+test_sets="test devel"
+
+slu_config=conf/tuning/train_asr_no_pretrain.yaml
```
Comment (ftshijt) [inline egs2/slue-voxpopuli/slu1/run.sh:12]: Consider make it standardize by setting conf/train.yaml

File: egs2/slue-voxpopuli/slu1/conf/tuning/train_asr_vq_apc.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+specaug: specaug
```
Comment (ftshijt) [inline egs2/slue-voxpopuli/slu1/conf/tuning/train_asr_vq_apc.yaml:1]: It seems to me that the change of many configs are quite similar, do we want to keep all of them? (I'm ok with it if that's for public benchmark, but ideally, it would be better if we could summarize them into fewer configs, which is also easier for users)

**Review by ftshijt — COMMENTED** [review]
Thanks for the update! Looks very good to me. Two minor comments

File: egs2/slue-voxpopuli/slu1/run.sh (hunk 1 above)
Comment (siddhu001) [reply egs2/slue-voxpopuli/slu1/run.sh]: Done!

File: egs2/slue-voxpopuli/slu1/conf/tuning/train_asr_vq_apc.yaml (hunk 2 above)
Comment (siddhu001) [reply egs2/slue-voxpopuli/slu1/conf/tuning/train_asr_vq_apc.yaml:1]: Yes, I considered it, but similar configs (like the ones using pre-trained ASR) have different sets of hyperparameters (like warmup steps, learning rate) selected based on validation performance. Hence, grouping similar configs would reduce reproducibility, so I think it is better to keep all of the configs.

**Comment by siddhu001** [comment]
@sw005320 I think this PR is ready to be merged!

File: egs2/slue-voxpopuli/slu1/README.md
Comment (sw005320) [inline egs2/slue-voxpopuli/slu1/README.md:1]: The table is not correctly displayed.
Can you confirm it?
image

Comment (siddhu001) [reply egs2/slue-voxpopuli/slu1/README.md:1]: Thanks for catching this! I have fixed this now!
